server/views/pages/npr.html

```diff
@@ -55,16 +44,6 @@ <h1 id="title" class="govuk-hub-media-player__title govuk-heading-xl govuk-body
     >
       <source src="{{ data.media }}" type="audio/ogg"/>
     </audio>
```

Comment:
`govuk-hub-on-black-feedback` class not imported  as it is not using the `mediaTemplate.njk`